packages/react-import-remote/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@shopify/react-import-remote",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "Provides a utility function and React component to asyncronously load scripts and extract the newly-added global",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "prepublishOnly": "yarn run build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "author": "Shopify Inc.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/quilt.git"
+  },
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
+  },
+  "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-import-remote/README.md",
+  "dependencies": {
+    "@shopify/react-preconnect": "^0.1.0"
+  },
+  "devDependencies": {
+    "@shopify/javascript-utilities": "^2.1.0",
```

Comment:
@TheMallen what is our plan around this? I noticed you used `lodash-decorators` directly in `react-form-state`.